packages/store/src/reactive/text.ts

```diff
@@ -20,6 +20,14 @@ export class Text {
     if (typeof input === 'string') {
       this._yText = new Y.Text(input);
```

Comment:
Should we also filter the normal string?

Reply:
Done